WireAPI/Sources/WireAPI/Models/Network/TLSVersion.swift

```diff
@@ -30,6 +30,23 @@ public enum TLSVersion {
 
     case v1_3
 
+    public static func minVersionFrom(_ string: String?) -> TLSVersion {
```

Comment:
I am not sure to see how this is related to this PR, is it for something further

Reply:
The APIService needs a `WireAPI.TLSVersion`.
The `AuthenticatedSessionFactory` has one, but as a String.
The `WireAPI.TLSVersion` had no way of being instantiated using a String.

So Monday I asked if I should port the `WireTransport.TLSVersion` init logic to `WireAPI.TLSVersion`, and was told I indeed should.